splunk/resource_splunk_configs_conf_test.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 const newConfigsConf = `
-resource "splunk_configs_conf" "tftest-stanza" {
-	name = "tftest/tftest_stanza"
+resource "splunk_configs_conf" "tf_test-stanza" {
+	name = "tf_test/tftest_stanza"
```

Comment:
The resource name doesn't need to change, and changing it causes the test to fail unless an additional change is made.

```suggestion
resource "splunk_configs_conf" "tftest-stanza" {
	name = "tf_test/tftest_stanza"
```